apps/ui/src/components/page-builder/components/BasicImage.tsx

```diff
@@ -43,25 +45,27 @@ export function BasicImage({
     width:
       forcedSizes?.width ??
       component?.width ??
-      media?.data?.attributes?.width ??
-      fallbackSizes?.width,
+      media?.width ??
+      fallbackSizes?.width ??
+      undefined,
 
     height:
       forcedSizes?.height ??
       component?.height ??
-      media?.data?.attributes?.height ??
-      fallbackSizes?.height,
+      media?.height ??
+      fallbackSizes?.height ??
+      undefined,
   }
 
-  const attributes = media?.data?.attributes
+  const attributes = media
 
   return (
     <ImageComp
       style={{ ...sizes, ...imgProps.style }}
       className={className}
       // @ts-expect-error - src needs to be typed in better way
       src={attributes?.url}
-      fallbackSrc={component?.fallbackSrc}
+      fallbackSrc={component?.fallbackSrc ?? undefined}
```

Comment:
Is it necessary to be explicit here? The optional chaining should return undefined implicitly, even if `component` is null.

Reply:
The main reason is that in v5 the inferred types of frontend attributes have `... | undefined | null` type always. In v4 it was `... | undefined`. So it breaks things in many places where we expected just `undefined`